CHANGELOG.md

```diff
@@ -125,7 +125,7 @@ Generated on 2024-01-31
 |[#10123](https://github.com/NVIDIA/spark-rapids/pull/10123)|Change version to v23.12.1 [skip ci]|
 |[#10122](https://github.com/NVIDIA/spark-rapids/pull/10122)|Init changelog for v23.12.1 [skip ci]|
 |[#10121](https://github.com/NVIDIA/spark-rapids/pull/10121)|[DOC] update download page for db hot fix  [skip ci]|
-|[#10116](https://github.com/NVIDIA/spark-rapids/pull/10116)|Upgrade to 23.12.1|
+|[#10116](https://github.com/NVIDIA/spark-rapids/pull/10116)|Upgrade to 23.12.1-SNPASHOT|
```

Comment:
Typo that's not in branch-24.02?  Curious how this change happened since it should just be a merge.

Reply:
Good catch! thanks @jlowe 

I found the root cause, that's my fault:

On dev branch-24.02 it is `23.12.1-SNAPSHOT` :  https://github.com/NVIDIA/spark-rapids/blob/branch-24.02/CHANGELOG.md


But, on main branch, I replaced all `23.12.1-SNAPSHOT` to `23.12.1`, where I should NOT modify CHANGELOG.md file :
https://github.com/NVIDIA/spark-rapids/commit/e83c1c1fdd70568cdbdb3ff13d43f7e5f6cb8ae5


So we should keep `23.12.1-SNPASHOT` here, to make it the same as what dev branch is.

Reply:
The dev branch does not have 23.12.1-SNPASHOT, it has 23.12.1-SNAPSHOT.  I'm curious how SNAPSHOT became SNPASHOT.

Reply:
Aha, sorry, this is a typo when I fixed merge conflicts to main from dev branch, let me fix it